src/org/usfirst/frc4904/sovereignty/FusibleNavX.java

```diff
@@ -0,0 +1,31 @@
+package org.usfirst.frc4904.sovereignty;
+
+
+import org.usfirst.frc4904.standard.custom.sensors.NavX;
+import edu.wpi.first.wpilibj.SerialPort.Port;
+
+public class FusibleNavX extends NavX implements Fusible<Double> {
```

Comment:
I agree. I added it like this for the sake of not making a branch on Standard. That's why it's in Sovereignty (all the stuff I put in here is standard quality for the most part and doesn't depend on anything in the yearly code.)